torchdata/nodes/map.py

```diff
@@ -272,6 +281,77 @@ def _shutdown(self):
                     t.join(timeout=QUEUE_TIMEOUT * 5)
 
 
+class _ParallelMapperImpl(BaseNode[T]):
+    """This class implements _ParallelMapperIter as a BaseNode, allowing it
```

Comment:
Nit: This class implements _ParallelMapperIter and _InlineMapperIter as a BaseNode, ....

Reply:
good catch!